Vagrant installer: flynn command not found

seems that `flynn` must be installed prior the execution of vagrant installer

```
Connection to 127.0.0.1 closed.
make[2]: uscita dalla directory "/home/ale/flynn/demo"
bash: riga 1: flynn: comando non trovato
Makefile:35: set di istruzioni per l'obiettivo "bootstrap" non riuscito
make[1]: *** [bootstrap] Errore 127
make[1]: uscita dalla directory "/home/ale/flynn/demo"
Makefile:22: set di istruzioni per l'obiettivo "init" non riuscito
make: *** [init] Errore 2
```

i've destroyed the vagrant VM, manually installed the `flynn` cli and started again the vagrant installed and everything went ok

So, in docs, you should say that `flynn` cli must be installed prior running `make init` or `make init` should install the cli during the process

--- a/demo/Makefile
+++ b/demo/Makefile
@@ -14,7 +14,7 @@ CLI_INSTALL_BIN := /usr/local/bin/flynn
 VAGRANT_ENV := CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" SSH_AUTH_SOCK="" FLAG_BOOTSTRAP=false
 
 
-.PHONY: init install-cli bootstrap up down halt update ssh reset token destroy dashboard ssh-cmd
+.PHONY: init bootstrap up down halt update ssh reset token destroy dashboard ssh-cmd
 
 
 # Initialize a new cluster
@@ -24,14 +24,14 @@ init: update
 	@printf "\nFlynn is now running in the VM.\nOpen dashboard: make dashboard\n\n"
 
 # Install flynn cli on host
-install-cli:
+$(CLI_INSTALL_BIN):
 	curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >${CLI_INSTALL_BIN}
 	chmod +x ${CLI_INSTALL_BIN}
 
 # Bootstrap the cluster
 # This should only be called once per cluster
 # Runs bootstrap in VM and cluster add on host
-bootstrap:
+bootstrap: $(CLI_INSTALL_BIN)
 	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee bootstrap.log
 	@(grep '^flynn cluster add .*$$' bootstrap.log || echo exit 1) | sed "s/ default / ${CLUSTER} /" | sed "s/$$(printf '\r')\$$/ --force --default/" | bash
 	@rm bootstrap.log
@@ -64,12 +64,12 @@ reset:
 
 # Destroy VM
 # Run this to delete the VM and free up disk space
-destroy:
+destroy: $(CLI_INSTALL_BIN)
 	${VAGRANT_ENV} vagrant destroy -f
 	flynn cluster remove ${CLUSTER}
 
 # Get token for API and dashboard
-token:
+token: $(CLI_INSTALL_BIN)
 	@flynn -a dashboard -c ${CLUSTER} env get LOGIN_TOKEN
 
 # Open dashboard in browser

